### ISSUE TITLE
[Bug] AP patched ROM of Black not uploading

### ISSUE THREAD

**Issue by lemollie** [Ajarmar/universal-pokemon-randomizer-zx#549]
**Describe the bug**
Pokemon Black ROM AP patched via DS-Scene ROM Tool will not upload for randomization
[error_2022-09-30-21-15-46.txt](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/files/9689316/error_2022-09-30-21-15-46.txt)

**To Reproduce**
1. Attempt to upload the following ROM: [https://drive.google.com/file/d/13bdqAar4tMYk6Zn44nLUE35LhOW0x6jN/view?usp=sharing](url)

**Expected behavior**
The AP patched ROM uploading and randomizing

**System:**
 - Operating System: Windows 10 Home (Ver. 1909, OS Build 18363.1556)
 - Java Version: Newest
 - Randomizer Version: Newest

**Additional context**
AP patching the ROM in an attempt to bypass antipiracy measures upon recommendation

**Comment by Ajarmar** [comment]
Why do you need to AP patch the ROM?

**Comment by lemollie** [comment]
> Why do you need to AP patch the ROM?

I'm trying to play on actual hardware and antipiracy activated with my original randomized ROM. The dev of TWLMenu++ recommended I AP patch a ROM then randomize that one, and as of today this method still leaves me with an error.

**Comment by Ajarmar** [comment]
Okay, I see. I checked your error log and it seems like you aren't using our release of the randomizer. I'm not sure if our release would work, but can you try downloading the randomizer from [here](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/releases/tag/v4.5.1) and see if it works?

**Comment by lemollie** [comment]
Looks like the error is occurring because of the fact it's AP patched. I tried randomizing a save on the version you provided and antipiracy still activated as well. Do you think this has something to do with my hardware?

**Comment by ShinyTurtwig** [comment]
I don’t know if i was supposed to make a new post since it’s a similar sort of issue, but i have an AP patched rom of Pokémon Black as well. And after the AP patch it says the rom is unofficial but i put the rom without the patch it says it’s official.

**Comment by Ajarmar** [comment]
@lemollie Do you mean that you randomized a regular ROM or a ROM that had already been AP patched? Randomizing a regular ROM and trying to use that randomized ROM in TWLMenu++ will activate antipiracy, this is a known issue that is more on the side of TWLMenu++ than on our side. Does it work if you randomize a ROM that has already been AP patched?

@ShinyTurtwig The randomizer detects that the ROM has been modified after the AP patch and that's why it says it's not an official ROM. What happens if you try to randomize it anyway, does it work?

**Comment by ShinyTurtwig** [comment]
@Ajarmar Yeah it does work, but it said it there could be issues, so that’s why I asked here.

**Comment by lemollie** [comment]
@Ajarmar I'm randomizing a ROM that has already been AP patched, sorry for the confusion. I'm still using the ROM provided in the initial post, if that helps.
Here's the error I'm getting from your release of the randomizer you recommended.

[error_2023-01-16-21-37-45.txt](https://github.com/Ajarmar/universal-pokemon-randomizer-zx/files/10430631/error_2023-01-16-21-37-45.txt)
